packages/cli/lib/projects.js

```diff
@@ -173,6 +173,10 @@ const pollBuildStatus = async (accountId, name, buildId) => {
 
       if (Object.keys(spinnies.spinners).length) {
```

Comment:
I was poking around the spinnies npm page (I'm new to that package), and I saw that there's a `hasActiveSpinners()` util. Could that replace the `Object.keys(spinnies.spinners).length` check?

Reply:
Good catch! I'll try swapping that out.